2018-05-15T08:24:32.474Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '009' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:24:32.475Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '009' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '009' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T08:24:33.105Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:24:33.106Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:24:33.109Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:24:33.109Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:24:33.109Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:24:33.109Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:24:33.114Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'tech@techcombank.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:24:33.114Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'tech@techcombank.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'tech@techcombank.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:24:33.117Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper@shipperltd.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:24:33.117Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper@shipperltd.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper@shipperltd.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:29:12.854Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:29:12.854Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T08:29:12.854Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-05-15T08:30:37.526Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:30:37.527Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T08:30:37.527Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-05-15T08:31:10.442Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:31:10.442Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T08:31:10.442Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-05-15T08:31:58.670Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:31:58.670Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T08:31:58.670Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-05-15T08:32:31.999Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/easymoney@easymoneyinc.com"}}$
2018-05-15T08:33:52.740Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#9b3fb6950f87fce9b47d100c25fd6162d4ed58969aa2de833de4d10c2724b570')
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:33:52.740Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#9b3fb6950f87fce9b47d100c25fd6162d4ed58969aa2de833de4d10c2724b570')\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#9b3fb6950f87fce9b47d100c25fd6162d4ed58969aa2de833de4d10c2724b570')\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:41:30.790Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#6ce4fc68ab1baac21e250dc680820fbf718fc27b23580f04731932d92985332d')
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T08:41:30.790Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#6ce4fc68ab1baac21e250dc680820fbf718fc27b23580f04731932d92985332d')\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#6ce4fc68ab1baac21e250dc680820fbf718fc27b23580f04731932d92985332d')\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T08:41:43.388Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/easymoney@easymoneyinc.com"}}$
2018-05-15T09:03:22.012Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/easymoney@easymoneyinc.com"}}$
2018-05-15T09:03:55.598Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/easymoney@easymoneyinc.com"}}$
2018-05-15T09:14:43.852Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#30ea436b19c6070121612a0278bbf2e1327c6d26dbeaeb473a4c1d081b19a2ea')
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:14:43.852Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#30ea436b19c6070121612a0278bbf2e1327c6d26dbeaeb473a4c1d081b19a2ea')\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#30ea436b19c6070121612a0278bbf2e1327c6d26dbeaeb473a4c1d081b19a2ea')\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:16:43.892Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#ba261fe213430b0b73161cf215cb01b3dc6790f69bccb020d2746c2a5ffec19c')
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:16:43.892Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#ba261fe213430b0b73161cf215cb01b3dc6790f69bccb020d2746c2a5ffec19c')\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Participant 'org.acme.Z2BTestNetwork.Provider#phongvu@phongvu.com' does not have 'CREATE' access to resource 'org.acme.Z2BTestNetwork.RequestPayment#ba261fe213430b0b73161cf215cb01b3dc6790f69bccb020d2746c2a5ffec19c')\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:26:25.996Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/easymoney@easymoneyinc.com"}}$
2018-05-15T09:28:12.290Z ERROR   HLFConnection           :createIdentity()         Register request failed trying to create identity {"0":{"stack":["{Error}fabric-ca request register failed with errors [[{\"code\":0,\"message\":\"Identity 'tech@techcombank.com' is already registered\"}]] Error: fabric-ca request register failed with errors [[{\"code\":0,\"message\":\"Identity 'tech@techcombank.com' is already registered\"}]]","    at IncomingMessage.<anonymous> (/home/damsen2/Chapter10/node_modules/fabric-ca-client/lib/FabricCAClientImpl.js:592:22)","    at emitNone (events.js:111:20)","    at IncomingMessage.emit (events.js:208:7)","    at endReadableNT (_stream_readable.js:1064:12)","    at _combinedTickCallback (internal/process/next_tick.js:138:11)","    at process._tickCallback (internal/process/next_tick.js:180:9)"]}}$
2018-05-15T09:30:30.084Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'easymoney@easymoneyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.FinanceCo' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:30:30.084Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'easymoney@easymoneyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.FinanceCo' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'easymoney@easymoneyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.FinanceCo' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T09:30:30.611Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:30:30.611Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:30:30.646Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:30:30.646Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:30:30.662Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper@shipperltd.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:30:30.662Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper@shipperltd.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper@shipperltd.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:30:30.684Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:30:30.684Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:30:30.881Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:30:30.881Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T09:30:47.222Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/easymoney@easymoneyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/easymoney@easymoneyinc.com"}}$
2018-05-15T09:31:33.585Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T09:31:33.585Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T09:31:33.585Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
